spec/services/search/search_service_spec.rb

```diff
@@ -11,6 +11,13 @@
       end
     end
 
+    context 'given the search engine is bing' do
+      it 'returns instance of BingSearchService' do
+        search_service = described_class.new(search_engine: 'bing', keyword: 'ruby')
+        expect(search_service).to be_a(Search::BingSearchService)
```

Comment:
Could you add a line break between Act and Assert parts? 🙏 The same for the above test